packages/console/src/consts/subscriptions.ts

```diff
@@ -18,6 +18,8 @@ export const tenantMembersAddOnUnitPrice = 8;
 export const mfaAddOnUnitPrice = 48;
 export const enterpriseSsoAddOnUnitPrice = 48;
 export const organizationAddOnUnitPrice = 48;
+export const tokenAddOnUnitPrice = 80;
+export const hooksAddOnUnitPrice = 2;
```

Comment:
Can we put these constants in the Cloud and fetched from the quota/subscription api, so that we can keep it SSOT?

Reply:
This is a TODO, I agree with you.